dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -23,10 +23,12 @@ class SentryStackFrame {
     this.symbolAddr,
     this.instructionAddr,
     this.rawFunction,
+    this.stackStart,
+    this.symbol,
     List<int>? framesOmitted,
     List<String>? preContext,
     List<String>? postContext,
-    Map<String, String>? vars,
+    Map<String, dynamic>? vars,
```

Comment:
Not a breaking change because String -> dynamic right.

Reply:
It shouldn't cause any issues since it now accepts even more types. Just reading out of it could lead to issues, but I guess that's not a typical usage for those values.